diff --git a/node_modules/next/dist/build/webpack/plugins/minify-webpack-plugin/src/index.js b/node_modules/next/dist/build/webpack/plugins/minify-webpack-plugin/src/index.js
index 663116d..dd0726e 100644
--- a/node_modules/next/dist/build/webpack/plugins/minify-webpack-plugin/src/index.js
+++ b/node_modules/next/dist/build/webpack/plugins/minify-webpack-plugin/src/index.js
@@ -17,13 +17,14 @@ function _interop_require_default(obj) {
     };
 }
 function buildError(error, file) {
+    const WebpackError = _webpack.webpack.WebpackError;
     if (error.line) {
-        return new _webpack.WebpackError(`${file} from Minifier\n${error.message} [${file}:${error.line},${error.col}]${error.stack ? `\n${error.stack.split('\n').slice(1).join('\n')}` : ''}`);
+        return new WebpackError(`${file} from Minifier\n${error.message} [${file}:${error.line},${error.col}]${error.stack ? `\n${error.stack.split('\n').slice(1).join('\n')}` : ''}`);
     }
     if (error.stack) {
-        return new _webpack.WebpackError(`${file} from Minifier\n${error.message}\n${error.stack}`);
+        return new WebpackError(`${file} from Minifier\n${error.message}\n${error.stack}`);
     }
-    return new _webpack.WebpackError(`${file} from Minifier\n${error.message}`);
+    return new WebpackError(`${file} from Minifier\n${error.message}`);
 }
 const debugMinify = process.env.NEXT_DEBUG_MINIFY;
 class MinifyPlugin {
